js/editor.js

```diff
@@ -145,8 +148,7 @@ class LoadSketch {
     }
 
     async download(url) {
-        this.goOnline();
-        return await firebase.database().ref(url).once('value');
+        return await this.restGet(url);
```

Comment:
Since there is no online / offline anymore, I would just convert the LoadSketch   class into a simple function  that gets passed (so no `new LoadSketch` anymore).